### DISCUSSION TITLE
The buttons in the navigation bar on the webpage have disappeared.

### DISCUSSION THREAD

**Discussion by ahdfiuhfo** [ClassicOldSong/Apollo#102] [Q&A] [answered]
![屏幕截图_10-11-2024_171815_localhost](https://github.com/user-attachments/assets/009992c7-231d-4382-91be-fe073be127b1)
Please help me!

**Comment by ClassicOldSong** [comment, marked as answer]
?????
How could this even happen?

Could you try upgrading to the latest version and see if it comes back?

Or could you change a browser and see if it's back? Might be some plugins removed those buttons.

**Reply by ahdfiuhfo** [reply]
Thank you! I used a new browser, and the issue is disappear. However, I still don't know which plugin was affecting the normal functioning of the webpage.

**Reply by ClassicOldSong** [reply]
Might be some ad blocking plugins.
